tests/travis/install-db2.sh

```diff
@@ -4,17 +4,20 @@ set -ex
 
 echo Setting up IBM DB2
 
-sudo docker pull ibmcom/db2express-c:10.5.0.5-3.10.0
-sudo docker run \
-    -d \
+sudo docker pull ibmcom/db2
+docker run \
+    -itd \
+    --name db2 \
+    --privileged=true \
     -p 50000:50000 \
-    -e DB2INST1_PASSWORD=Doctrine2018 \
     -e LICENSE=accept \
-    --name db2 \
-    ibmcom/db2express-c:10.5.0.5-3.10.0 \
-    db2start
+    -e DB2INST1_PASSWORD=Doctrine2018 \
+    -e DBNAME=doctrine \
+    ibmcom/db2
+
+while ! sudo docker exec db2 su - db2inst1 -c 'which db2 2> /dev/null'; do sleep 1; done
 
-sleep 15
+while ! sudo docker exec db2 su - db2inst1 -c 'db2start'; do sleep 1; done
```

Comment:
Won't the container start automatically?

> To confirm Db2 container is ready, in the logs we will see the message **Setup has completed**.

Maybe we could try the above?

Reply:
Yes, it will. Removed it and changed it to wait for db2 to start.